language/move-binary-format/src/file_format.rs

```diff
@@ -864,6 +858,12 @@ pub enum SignatureToken {
     MutableReference(Box<SignatureToken>),
     /// Type parameter.
     TypeParameter(TypeParameterIndex),
+    /// Unsigned integers, 16 bits length.
```

Comment:
For these out-of-order variants, I think it's useful to have a comment like `// added in bytecode version v6, do not reorder!` so future folks understand why it works this way.